### ISSUE TITLE
Generate Input Action Names

### ISSUE THREAD

**Issue by PadraigK** [PadraigK/SceneGen#7]
Inputs are defined in project settings, and could easily by generated by SceneGen too:

<img width="880" alt="Screenshot 2023-10-13 at 3 16 46 PM" src="https://github.com/PadraigK/SceneGen/assets/466150/df5c43b6-4c0b-4b26-b8cb-336afe6c2606">

**Comment by PadraigK** [comment]
This turned out to be more complicated than I guessed:
1. There's no method on `ProjectSettings` to list all settings, just accessors to look them up by name
2. A name would be `Input/left` but `left` is user defined, so we can't know it ahead of time. 
3. I was about to propose it be added when I found this: https://github.com/godotengine/godot-proposals/discussions/4794 which points to using `getPropertyList()` function on `Object` (which ProjectSettings subclasses)
4. But there's an issue in SwiftGodot preventing the ProjectSettings singleton from being reachable:
https://github.com/migueldeicaza/SwiftGodot/issues/171

Once that issue is fixed, it should be a matter of digging into `getPropertyList` to find any properties prefixed with `input/` and then looking them up using the ProjectSettings methods.
